docs/dev/table/connectors/jdbc.md

```diff
@@ -148,6 +148,13 @@ Connector Options
       <td>String</td>
       <td>The JDBC password.</td>
     </tr>
+    <tr>
+      <td><h5>connection.max-retry-timeout</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">60s</td>
+      <td>Duration</td>
+      <td>Maximum timeout between retries.</td>
```

Comment:
Would be better to mention that the timeout should be in second granularity and shouldn't be smaller than 1 second.